packages/@glimmerx/ssr/tests/index.ts

```diff
@@ -2,7 +2,35 @@ import renderTests from '@glimmerx/core/tests/render-tests';
 import { Constructor } from '@glimmerx/core';
 import Component from '@glimmerx/component';
 import { renderToString, RenderOptions } from '..';
+import { setComponentTemplate } from '@glimmerx/core';
+import { compileTemplate } from '@glimmerx/core/tests/utils';
+import HTMLSerializer from '@simple-dom/serializer';
+import voidMap from '@simple-dom/void-map';
+import { SerializableNode } from '@simple-dom/interface';
 
 renderTests('@glimmer/ssr', async (component: Constructor<Component>, options: RenderOptions) => {
   return await renderToString(component, options);
+});
+
+QUnit.module('@glimmer/ssr rendering', () => {
```

Comment:
Can we move this into its own test file rather than the index

Reply:
Yes, I can. I thought about it, but I wasn't sure what to name it. Moving it out helps clarify the dependencies for the test.